completions/_git-forgit

```diff
@@ -99,3 +97,28 @@ _git-forgit() {
 		;;
 	esac
 }
+
+# Check if forgit plugin is loaded
+if (( $+functions[forgit::add] )); then
```

Comment:
Now that we are no longer sourcing this, we no longer need to check if the plugin is loaded. Compsys will not execute this file in this case.